guides/wayback-integration-guide.rst

```diff
@@ -156,6 +168,14 @@ Open your `wct-core.properties` file and make the following changes. (`wct-core.
     qualityReviewToolController.enableAccessTool=true
     qualityReviewToolController.archiveUrl=http://localhost:8080/wayback/*/
 
+Using Multiple Review Tools in WCT
+==================================
+
+Within the Target Summary for the harvest you will have options for different Quality Review Tools.  There will be a link to Review in Access Tool plus other links to other archives which you can specify the name of.  All of these links are configurable via wct-core.properties.  
+
+-   Review in Access Tool uses the value set in harvestResourceUrlMapper.urlMap
+-   qualityReviewToolController.archiveName uses the value set in qualityReviewToolController.archiveUrl
+-   qualityReviewToolController.archive.alternative.name uses the value set in qualityReviewToolController.archive.alternative
```

Comment:
I think this part could be clearer, perhaps a screenshot of the UI could help make it obvious what these three properties refer to.

Reply:
I've added an image which shows which config affects which links.